source/common/ssl/tls_certificate_secret_impl.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/api/v2/auth/cert.pb.h"
+#include "envoy/secret/secret.h"
+
+namespace Envoy {
+namespace Ssl {
+
+class TlsCertificateSecretImpl : public Secret::TlsCertificateSecret {
+public:
+  TlsCertificateSecretImpl(const std::string& name,
+                           const envoy::api::v2::auth::TlsCertificate& config);
+  virtual ~TlsCertificateSecretImpl() {}
+
+  const std::string& name() const override { return name_; }
```

Comment:
I don't believe this is ever used? It's just a lookup key for `SecretManager`, so it doesn't have to be part of the value itself.

Reply:
Removed